### PR TITLE
advanced sass and folder structure

### PR DESCRIPTION
**Pull request by nicopixeldev**

### CODE REVIEW

**Review by quinwacca — APPROVED** [review]
Awesome work!

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,9 @@
+.gitignore
```
Comment (maryammdot) [inline .gitignore]: You shouldn't modify this file!!! Try to undo the change 🥇
